tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+# The tutorial template
+
+The tutorial template includes:
+
+* Placeholder text that you can replace with your own text. This text is indicated in {curly braces}.
+* Writing instructions, also in {curly braces}.
+
+## Why do I need tutorials?
+
+Tutorials are integral to helping your users become acquainted with your product. They are learning-oriented, which means that they teach users how to do a practical task, like debugging a JavaScript application. 
+
+Tutorials are often confused with how-to guides because they help users achieve a specific goal. However, how-to guides assume that your users have some experience with your product or are familiar with certain concepts. 
+
+In comparison, tutorials give your users hands-on experience with your product so that they can understand key concepts about your product in a more practical context. Hands-on learning often helps learners gain a deeper understanding of the product by helping them see how they can use a tool to accomplish a task they care about.
+
+## Helpful tips when writing your tutorial 
+
+* Keep your tutorial focused. If needed, include links to other pieces of documentation that explain concepts or terms in more depth.
+* Lengthy tutorials can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the tutorial.
+* If users will copy and paste code samples, make sure they include the following:
+  * Any required `import` or `using` statements
+  * Code comments that explain what the code does 
+
+## About the tutorial template 
+
+This tutorial can be used in one of two ways: 
+* For providing a walkthrough of a particular feature.
+* Walking users through completing a small project. 
+
+Ideally, your users already completed a quickstart or "getting started" tutorial and have a general understanding of your product.  
```

Comment:
**Suggestion (phrasing, non-blocking):** For clarity, add "...and have a general understanding of your product before they begin your tutorial."